docs/resources/rds_database.md

```diff
@@ -0,0 +1,54 @@
+---
+subcategory: "Relational Database Service (RDS)"
+---
+
+# huaweicloud_rds_database
+
+Manages RDS Mysql database resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "instance_id" {}
+
+resource "huaweicloud_rds_database" "test" {
+  instance_id   = huaweicloud_rds_instance.test.id
+  name          = var.instance_id
+  character_set = "utf8"
+  description   = "test database"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) The region in which to create the RDS database resource. If omitted, the
+  provider-level region will be used. Changing this creates a new resource.
+
+* `instance_id` - (Required, String, ForceNew) Specifies the RDS instance id. Changing this will create a new resource.
```

Comment:
suggest to make proper noun abbreviations be represented in all capitals, s.g. `RDS`, `ID`.

Reply:
fixed